lua/terrortown/lang/ru.lua

```diff
@@ -1,13 +1,8 @@
 -- Russian language strings
 
--- This localization file is being moderated and constantly updated by Satton(RU).
--- Please, in case you are making some changes ping @Satton2 on GitHub, contact him on Steam (STEAM_0:0:85981967) or Discord (Satton(RU)#5794).
-
```

Comment:
I would like the tool to not change such sections, can we introduce something to prevent it from changing certain sections like these? Maybe something like `-- %keep%`

Reply:
In theory it keeps the whole header (until L.__alias is reached), I guess it made a mistake here, hm

Reply:
I think I removed this manually by accident because I thought it was twice the same